### PR TITLE
I changed it to Atilgan List

### PR DESCRIPTION
**Pull request by alpatilgan**
I changed the title to AtilganList is that a fine title?

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
```
Comment (adilatilganallocadia) [inline src/app/app.component.html]: Why did you remove div? Maybe you want to put it back we will need it.

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,9 @@
 import { Component } from '@angular/core';
-
```
Comment (adilatilganallocadia) [inline src/app/app.component.ts]: Please do not remove space here, We usually need a space between imports and component declaration. 
It is more readable this way.

File: src/app/app.component.html (hunk 1 above)
Comment (alpatilgan) [reply src/app/app.component.html]: I don't even know what that is?

File: src/app/app.component.ts (hunk 2 above)
Comment (alpatilgan) [reply src/app/app.component.ts]: There is no space there on my screen?!

**Comment by adilatilganallocadia** [comment]
Wrong one please read carefully .

**Review by adilatilganallocadia — APPROVED** [review]
LGTM
